FightPandemics/Features/Post/CreatePost/CreatePostFooter.swift

```diff
@@ -43,7 +43,7 @@ class CreatePostFooter: UIView {
     }
 
     private func setUpUpper () {
-        // seperator line
+        // separator line
```

Comment:
I deleted this!! upon your comments!! how come its still in code base

Reply:
it may have made its way back while i was resolving conflicts 🤷